src/tools/run-make-support/src/lib.rs

```diff
@@ -74,6 +74,12 @@ pub fn ar(inputs: &[impl AsRef<Path>], output_path: impl AsRef<Path>) {
     }
 }
 
+/// Check if target is linux-like.
+#[must_use]
+pub fn is_linux() -> bool {
+    target().contains("linux")
+}
```

Comment:
Suggestion: and since we would like to prefer to use `//@ ignore-linux`, could you please revert this change.